app/components/gh-file-uploader.js

```diff
@@ -74,6 +76,20 @@ export default Component.extend({
         }
     },
 
+    didReceiveAttrs() {
+        this._super(...arguments);
+        let accept = this.get('accept');
+        let extensions = this.get('extensions');
+
+        if (!isBlank(accept) && !isEmberArray(accept)) {
+            this.set('accept', accept.split(','));
```

Comment:
Hmm, this would result in updating the two-way binding. I was thinking something like `this._accept = accept.split(',')` - no need to use getters/setters as we're dealing with completely internal POJOs.